aws-glue-schema/aws-glue-schema.json

```
@@ -0,0 +1,173 @@
+                    "type": "string",
+                    "pattern": "arn:(aws|aws-us-gov|aws-cn):glue:.*"
+                }
+            }
+        },
+        "SchemaVersion": {
+            "type": "object",
+            "description": "Specify checkpoint version for update. This is only required to update the Compatibility.",
+            "properties": {
+                "IsLatest": {
+                    "description": "Indicates if the latest version needs to be updated.",
+                    "type": "boolean"
+                },
+                "VersionNumber": {
+                    "description": "Indicates the version number in the schema to update.",
+                    "type": "integer",
+                    "minimum": 1,
+                    "maximum": 100000
+                }
+            }
+        },
+        "Tag": {
+            "type": "object",
+            "properties": {
+                "Key": {
+                    "description": "A key to identify the tag.",
+                    "type": "string",
+                    "minLength": 1,
+                    "maxLength": 128
+                },
+                "Value": {
+                    "description": "Corresponding tag value for the key.",
+                    "type": "string",
+                    "minLength": 0,
+                    "maxLength": 256
+                }
+            },
+            "required": [
+                "Key",
+                "Value"
+            ],
+            "additionalProperties": false
+        }
+    },
+    "properties": {
+        "Arn": {
+            "description": "Amazon Resource Name for the Schema.",
+            "type": "string",
+            "pattern": "arn:(aws|aws-us-gov|aws-cn):glue:.*"
+        },
+        "Registry": {
+            "$ref": "#/definitions/Registry"
+        },
+        "Name": {
+            "description": "Name of the schema.",
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 255
+        },
+        "Description": {
+            "description": "A description of the schema. If description is not provided, there will not be any default value for this.",
+            "type": "string",
+            "minLength": 0,
+            "maxLength": 1000
+        },
+        "DataFormat": {
+            "description": "Data format name to use for the schema. Accepted values: 'AVRO'",
+            "type": "string",
+            "enum": ["AVRO"]
+        },
+        "Compatibility": {
+            "description": "Compatibility setting for the schema.",
+            "type": "string",
+            "enum": ["NONE", "DISABLED", "BACKWARD", "BACKWARD_ALL", "FORWARD", "FORWARD_ALL", "FULL", "FULL_ALL"]
+        },
+        "SchemaDefinition": {
+            "description": "Definition for the initial schema version in plain-text.",
+            "type": "string",
+            "minimum": 1,
+            "maximum": 170000
```

https://github.com/aws-cloudformation/cloudformation-cli/issues/414
https://json-schema.org/understanding-json-schema/reference/string.html#length
```suggestion
            "type": "string",
            "minLength": 1,
            "maxLength": 170000
```